de.wwwtech.ColorMate.json

```
@@ -0,0 +1,48 @@
+{
+  "app-id": "de.wwwtech.ColorMate",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.36",
```

Update runtime to 3.38
```suggestion
  "runtime-version": "3.38",
```